packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -96,7 +97,11 @@ export class SummarizerNode implements IRootSummarizerNode {
 		});
 	}
 
-	public startSummary(referenceSequenceNumber: number, summaryLogger: ITelemetryBaseLogger) {
+	public startSummary(
```

Comment:
Feel like adding a doc comment here?  e.g. explaining what this does relative to `summarize` and `completeSummary`?